app/actors/curation_concerns/base_actor.rb

```diff
@@ -11,17 +11,22 @@ def create(attributes)
       @cloud_resources = attributes.delete(:cloud_resources.to_s)
       apply_creation_data_to_curation_concern
       apply_save_data_to_curation_concern(attributes)
-      next_actor.create(attributes) && save
+      next_actor.create(attributes) && save && run_callbacks(:after_create_content)
```

Comment:
This isn't really "create_content".  That was for a binary deposit. This is "create metadata". Can we make a new callback type?